### PR TITLE
test boards and scripts + names and values export

### PR DESCRIPTION
**Pull request by 1am**
* Add two test board and schematic projects for easier testing + demonstration of usage of the script
* Export names and values diff so changes in their placement can be compared

### CODE REVIEW

**Comment by fxkr** [comment]
Nice, thank you!

I haven't used eagle-diff in a while, and never before with the current eagle8. Unfortunately, it seems the auto-alignment feature is broken. May I ask what version of eagle you're using, and whether the auto-alignment worked for you? Here's what I'm talking about:

image

Here's all of my output: https://paste.fxkr.net/9gfhn8kcqcua/

File: test/diff-test.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+python ../eagle-diff \
+    --eagle=/Applications/EAGLE-6.6.0/EAGLE.app/Contents/MacOS/EAGLE \
+    ./boards/led_buttons-1/led_buttons-1.sch \
+    ./boards/led_buttons-2/led_buttons-2.sch \
+    ./output
+
+# Diff from boards
+python ../eagle-diff \
+    --force \
+    --eagle=/Applications/EAGLE-6.6.0/EAGLE.app/Contents/MacOS/EAGLE \
```
Comment (fxkr) [inline test/diff-test.sh]: Might be a good idea not to hardcode magic values that are specific to your installation. But you could use e.g. --eagle=eagle to highlight that there is a --eagle switch, but otherwise leave it at its deafult.

File: eagle-diff [hunk 2]
```diff
@@ -275,9 +280,9 @@ def main(args):
     par.add_argument('--eagle', default="eagle",
         help="eagle binary to use (default: %(default)s)")
     par.add_argument('file1', type=os.path.abspath,
-        help="'old' version of the .brd file")
+        help="'old' version of the .brd or .sch file")
```
Comment (fxkr) [inline eagle-diff]: This is a completely unrelated change and should get its own commit

File: test/boards/led_buttons-1/led_buttons-1.pro [hunk 3]
```diff
@@ -0,0 +1,62 @@
+update=22/05/2015 07:44:53
+version=1
+last_client=kicad
```
Comment (fxkr) [inline test/boards/led_buttons-1/led_buttons-1.pro]: Someone secretly using kicad? ;-) Nice.

**Comment by fxkr** [comment]
So my review comments above are really just nitpicks. I'll just merge it as it is anyway, because it's not important for this small tool to do everything 100% clean, and these really really minor problems don't make anything worse.

Thanks for finding this tool helpful enough to contribute!

File: test/diff-test.sh (hunk 1 above)
Comment (karlp) [reply test/diff-test.sh]: --eagle=eagle is an _excellent_ suggestion.  I'm commenting to give kudos, but I'm _definitely_ stealing that idea for displaying the option, without necessarily using it!

File: test/boards/led_buttons-1/led_buttons-1.pro (hunk 3 above)
Comment (1am) [reply test/boards/led_buttons-1/led_buttons-1.pro]: Hah, using both. This file shouldn't be here. sorry
